include/seqan3/core/simd/detail/builtin_simd_intrinsics.hpp

```diff
@@ -0,0 +1,28 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2019, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2019, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+/*!\file
+ * \brief Provides intrinsics include for builtin simd.
+ * \author Marcel Ehrhardt <marcel.ehrhardt AT fu-berlin.de>
+ */
+
+#pragma once
+
+#include <seqan3/core/platform.hpp>
+
+#if __has_include(<x86intrin.h>)
+#include <x86intrin.h> // x86 intrinsics (linux)
+#endif
+
+#if __has_include(<intrin.h>)
+#include <intrin.h> // x86 intrinsics (windows)
+#endif
+
+#if defined(_MSC_VER) && defined(__AVX__) && !defined(__SSE4_1__) && !defined(__SSE4_2__)
```

Comment:
Please add a comment that windows might not define SSE if avx is available and that we need this to allow using sse also if avx is supported.